test/evergreen.yml

```diff
@@ -2805,14 +2806,16 @@ tasks:
         vars:
           # CC is set to the system default "clang" binary here as a workaround. Change it back
           # to mongodbtoolchain "clang" binary after BUILD-10248 fix.
-          configure_env_vars:
-            CCAS=/opt/mongodbtoolchain/v3/bin/gcc CC=/usr/bin/clang
-            CXX=/opt/mongodbtoolchain/v3/bin/clang++ PATH=/opt/mongodbtoolchain/v3/bin:$PATH
-            CFLAGS="-ggdb -fPIC -fsanitize=address -fno-omit-frame-pointer
-            -I/opt/mongodbtoolchain/v3/lib/gcc/ppc64le-mongodb-linux/8.2.0/include"
-            CXXFLAGS="-ggdb -fPIC -fsanitize=address -fno-omit-frame-pointer
-            -I/opt/mongodbtoolchain/v3/lib/gcc/ppc64le-mongodb-linux/8.2.0/include"
-          posix_configure_flags: --enable-diagnostic --with-builtins=lz4,snappy,zlib
+          posix_configure_flags:
+            -DHAVE_DIAGNOSTIC=1
+            -DHAVE_BUILTIN_EXTENSION_LZ4=1
```

Comment:
We need `-DWITH_PIC=1`

Reply:
Seems like we can copy and paste from line 2582 to 2589

Reply:
Its not really necessary, since its only building a shared target. Added it anyway in 8ac995589916edf337d5ec4c20bec8edf4a5abe3 to be consistent.